platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LocalLogoutServlet.java

```diff
@@ -102,4 +112,26 @@ private void deleteJSessionId(HttpServletResponse response) {
     cookie.setComment("EXPIRING COOKIE at " + System.currentTimeMillis());
     response.addCookie(cookie);
   }
+
+  private String getRedirectUriMessage() {
+    String redirectUriMessage = "";
+    if (Strings.isNotBlank(SystemBaseUrl.INTERNAL.getBaseUrl())
+        && Strings.isNotBlank(redirectUri)) {
+      try {
+        URIBuilder redirectUrlBuilder =
+            new URIBuilder(SystemBaseUrl.INTERNAL.getBaseUrl() + redirectUri);
+        redirectUriMessage =
+            redirectUrlBuilder == null ? "" : redirectUrlBuilder.build().toString();
+      } catch (URISyntaxException e) {
+        LOGGER.debug(
+            "Invalid URI of %s set for logout redirect uri. Users will not be redirected to %s upon logout.",
```

Comment:
this logging should be parameterized

Reply:
Yeah this is a format string, not a parameterized log string. By parameterized, we mean:
```
"Invalid URI of {} set for logout redirect uri. Users will not be redirected to {} upon logout."
```